src/systemctl/systemctl-list-units.c

```diff
@@ -760,3 +761,196 @@ int verb_list_timers(int argc, char *argv[], void *userdata) {
 
         return r;
 }
+
+struct automount_info {
+        const char* machine;
+        const char* id;
+        char* what;
+        char* where;
```

Comment:
We usually place `*` at the beginning of the variable name: `const char *machine`.

Reply:
Fixed.